### DISCUSSION TITLE
Allow PCSX-Redux users to check for updates manually

### DISCUSSION THREAD

**Discussion by ghost** [grumpycoders/pcsx-redux#1077] [Ideas]
In the Help menu add a new command "Check for Updates".
If there is no new updates then a message box appears:

> There is no new updates. Check again later.

Then click on the Okay button to close the message box and continue using PCSX-Redux.

If there are new updates then PCSX-Redux asks the user if he or she wants to update and if the user clicked on the "Yes" button then PCSX-Redux downloads the updates, installs them, rename old files so the user will be always be able to revert to the previous version if he or she wants and then reboot to use the new updates.

In the question dialog whether or not to update PCSX-Redux add a "What's new?" button. If the user clicks on this button then he or she will see what's new in this update, what this update changes, what this update adds, what this update removes and etc.

A possible, temporary, simple and easy workaround is to just reboot PCSX-Redux while the "Auto Update" is enabled to check for updates but be warned that all progress is lost if neither saved state nor saved to a memory card before this action.

**Comment by wheremyfoodat** [comment]
Partially superseded in #1079

**Reply by ghost** [reply]
> Partially superseded in #1079

If I understand you correctly "Enable Auto Update" supersedes "Check for Updates". Am I correct?

**Reply by wheremyfoodat** [reply]
> > Partially superseded in #1079
> 
> If I understand you correctly "Enable Auto Update" supersedes "Check for Updates". Am I correct?

Sort of, yeah. Things like presenting the user with a changelog and stuff would be nice to add in the future though.

**Reply by ghost** [reply]
> > > Partially superseded in #1079
> > 
> > 
> > If I understand you correctly "Enable Auto Update" supersedes "Check for Updates". Am I correct?
> 
> Sort of, yeah. Things like presenting the user with a changelog and stuff would be nice to add in the future though.

Agreed. You can give this feature request a low priority because of the already existing "Enable Auto Update" feature that can be used meanwhile if you want to.
